README.md

```diff
@@ -3,7 +3,7 @@
 [![MIT License](https://img.shields.io/badge/License-MIT-brightgreen.svg)](LICENSE)
 [![tests](https://github.com/envato/samson-deploy-buildkite-plugin/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/envato/samson-deploy-buildkite-plugin/actions/workflows/tests.yml)
 
-A [Buildkite plugin](https://buildkite.com/docs/agent/v3/plugins) that lets you send webhooks to Samson.
```

Comment:
Please correct me if I'm wrong.

In my understanding.

* Samson lists many webhooks for various other services, like Buildkite, Jenkins and GitHub in this page https://deploy.platform.envato.net/projects/author-finances/webhooks (maybe replace `author-finances` part with your own project name)
* When Buildkite pipeline runs, it prepares data like our repository code, then sends the data as payload to the above Samson webhook
* Then Samson will react to this incoming request, take the data and start doing deployment

So I think we don't really send the webhook to Samson from buildkite side, unless I'm confusing the jargons here?

Reply:
A couple of things, first be aware this is a public repo so anything we say is publicly visible

Samson provides "webhook URLs", the webhook is the payload we deliver as you describe.

Reply:
Yes, thanks for the reminder and confirmation.